graphql/resolvers.go

```diff
@@ -3030,13 +3030,27 @@ func (r *taskResolver) IsPerfPluginEnabled(ctx context.Context, obj *restModel.A
 				return false, err
 			}
 			for _, projectName := range perfPlugin.Projects {
-				if projectName == pRef.Id || projectName == pRef.Identifier {
-					return true, nil
+				if projectName != pRef.Id && projectName != pRef.Identifier {
+					return false, nil
 				}
```

Comment:
This logic doesn't look correct. 
The `pRef.Id` and `Identifier` are not always consistent with each other which is why we had the `or` statement. So we only needed to match on one of those values. 

When we return a value from a resolver the request ends so any code below will be unreachable. 

So if the first value checked in `perfPlugin.Projects` satisfies the conditional the resolver will return false and not allow any other elements to be checked.